psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -262,106 +254,74 @@ std::map<std::string, double> LibXCFunctional::query_libxc(const std::string& fu
 
     return params;
 }
-void LibXCFunctional::set_tweak(std::vector<double> values) {
-    bool failed = true;
+void LibXCFunctional::set_tweak(std::vector<double> values, bool quiet) {
```

Comment:
Should this function be tweaked to take in `std::map<std::string, double>`? It currently seems much less safe than the key/value counterpart.

Reply:
I now see the below, can we remove this signature entirely?

Reply:
I was going to leave it and add an upgrade helper for softer deprecation. I'll actually add the text now...